react-app/components/FileManager/index.jsx

```diff
@@ -39,11 +39,9 @@ const FileManager = (props) => {
     setDownloadFileErrorMessage(null);
     const { isStudioView } = xBlockContext;
     if(!isStudioView) {
-      /* change this as you need it */
-      sendTrackingLogEvent('edx.course.tool.accessed', {
-        course_id: 'courseId',
-        is_staff: 'administrator',
-        tool_name: 'analyticsId',
+      sendTrackingLogEvent('edunext.xblock.filesmanager.files.downloaded', {
```

Comment:
could you include more fields like user_id or username or email, current_date

Reply:
Done